client/components/marketing-survey/cancel-purchase-form/nextStep.js

```diff
@@ -0,0 +1,9 @@
+export default function nextStep( currentStep, steps ) {
+	const index = steps.indexOf( currentStep );
+
+	if ( index >= 0 && index < steps.length - 1 ) {
+		return steps[ index + 1 ];
+	}
+
+	return steps[ steps.length - 1 ];
+}
```

Comment:
please note that we don't have to make all this branching and instead could borrow a little math…

```js
/**
 * Returns the next item in a list after a given item
 *
 * If item is not found, return the first item
 * If item is at end, return the last item
 */
const nextAfter = ( item, list ) =>
	list[ Math.min(
		list.indexOf( item ) + 1,
		list.length - 1,
	) ];
```